batchflow/models/metrics/regression.py

```diff
@@ -108,6 +109,9 @@ def __getattr__(self, name):
     def mean_absolute_error(self):
         return np.average(np.abs(self.predictions - self.targets), axis=0, weights=self.weights)
 
+    def mean_absolute_percentage_error(self):
+        return np.average(np.abs((self.predictions - self.targets) / self.targets), axis=0, weights=self.weights) * 100
```

Comment:
What about division by zero?

Reply:
In this case `numpy` will raise the warning